ipysheet/sheet.py

```diff
@@ -35,6 +36,16 @@ class Cell(widgets.Widget):
     @traitlets.validate('value')
     def _validate_value(self, proposal):
         value = proposal['value']
+
+        original_value = value
+
+        # a pandas dataframe will hit this path first
+        if hasattr(value, "to_numpy"):
+            value = value.to_numpy()
+        # numpy arrays or pandas series will hit this path
+        if hasattr(value, "tolist"):
+            value = value.tolist()
+
```

Comment:
@maartenbreddels we want to remove that no? It will be done later in the serializer

Reply:
No we need it actually, for the shape checking